### PR TITLE
[#48261] Improving the "Troubleshooting" section of the admin guide for the Nextcloud integration

### PR DESCRIPTION
**Pull request by wielinde**
https://community.openproject.org/work_packages/48261

### CODE REVIEW

File: docs/system-admin-guide/integrations/nextcloud/README.md [hunk 1]
```diff
@@ -200,28 +200,70 @@ Once the file storage is added and enabled for projects, your users are able to
...
   ```bash
-  curl --location --request GET 'https://nextcloud.example.com/index.php/apps/integration_openproject/check-config' --header 'Authorization: foo'
-  
-  # If Nextcloud OpenProject Integration is integrated correctly the response should look like the following
-  
-  {"user_id":"","authorization_header":foo}
+  curl https://openproject.example.com
   ```
-  
+  In case the `curl` command above results in an error pay attention what it is telling you. Typical reasons for connection issues are misconfigured fire walls, proxies, or a bad TLS/SSL setup.
```
Comment (apfohl) [inline docs/system-admin-guide/integrations/nextcloud/README.md]: `fire walls` -> `firewalls`

File: docs/system-admin-guide/integrations/nextcloud/README.md [hunk 2]
```diff
@@ -200,28 +200,70 @@ Once the file storage is added and enabled for projects, your users are able to
...
+
+##### While using a self signed TLS/SSL certificate you receive "certificate verify failed"
+
+Some administrators setup OpenProject using a self signed TLS/SSL certificate with their own CA (certificate authority). The CA needs to be known on the Nextcloud server. On a Debian/Ubuntu based server, make sure you add the CA certificate for your OpenProject certificate to `/usr/local/share/ca-certificates` and run `sudo update-ca-certificates` afterwards. Then Nextcloud's PHP code should be able to verify your Nextcloud TLS/SSL certificate when emitting HTTPS requests to your Nextcloud server.
```
Comment (apfohl) [inline docs/system-admin-guide/integrations/nextcloud/README.md]: `...emitting HTTPS requests to your Nextcloud server.` -> `...emitting HTTPS requests to your OpenProject server.`

File: docs/system-admin-guide/integrations/nextcloud/README.md [hunk 3]
```diff
@@ -200,28 +200,70 @@ Once the file storage is added and enabled for projects, your users are able to
...
   
+  ```
   {"ocs":{"meta":{"status":"ok","statuscode":200,"message":"OK"},"data":{"version":{"major":24,"minor":0,"micro":6,"string":"24.0.6","edition":"","extendedSupport":false},"capabilities":{"bruteforce":{"delay":0},"metadataAvailable":{"size":["\/image\\\/.*\/"]},"theming":{"name":"Nextcloud","url":"https:\/\/nextcloud.com","slogan":"a safe home for all your data","color":"#0082c9","color-text":"#ffffff","color-element":"#0082c9","color-element-bright":"#0082c9","color-element-dark":"#0082c9","logo":"https:\/\/nextcloud.example.com\/nextcloud\/core\/img\/logo\/logo.svg?v=0","background":"https:\/\/nextcloud.example.com\/nextcloud\/core\/img\/background.png?v=0","background-plain":false,"background-default":true,"logoheader":"https:\/\/nextcloud.example.com\/nextcloud\/core\/img\/logo\/logo.svg?v=0","favicon":"https:\/\/nextcloud.example.com\/nextcloud\/core\/img\/logo\/logo.svg?v=0"}}}}}
   ```
+  If you do not get such a response check out what `curl` command above is telling you. Typical reasons for connection issues are misconfigured fire walls, proxies, or a bad TLS/SSL setup.
```
Comment (apfohl) [inline docs/system-admin-guide/integrations/nextcloud/README.md]: - "check out what `curl` command" -> "check out what the `curl` command"
- `fire walls` -> `firewalls`

**Review by apfohl — CHANGES_REQUESTED** [review]

File: docs/system-admin-guide/integrations/nextcloud/README.md (hunk 2 above)
Comment (wielinde) [reply docs/system-admin-guide/integrations/nextcloud/README.md]: Great catch! There is even another place in the sentence where "Nextcloud" needed to be replaced with "OpenProject".

**Comment by wielinde** [comment]
Thank you @apfohl for the review. Great catches! Please feel free to merge if you don't spot any other issue.

**Comment by apfohl** [comment]
@wielinde Could you please close the WP? I merged from my mobile phone ☺️.

**Comment by wielinde** [comment]
Sure! Thanks.

**Pull request merged**